pol-core/bscript/object.cpp

```diff
@@ -1984,7 +2065,7 @@ BObjectImp* BFunctionRef::call_method( const char* methodname, Executor& ex )
 
 bool BFunctionRef::validCall( const int id, Executor& ex, Instruction* inst ) const
 {
-  if ( id != MTH_CALL )
+  if ( id != -1 && id != MTH_CALL )
```

Comment:
Why not use for continuation also the MTH_CALL id. In the end you want to call it, so why add a special handling for it